src/os/filestore/FileStore.h

```diff
@@ -767,6 +767,8 @@ class FileStore : public JournalingObjectStore,
 
   virtual int apply_layout_settings(const coll_t &cid, int target_level);
 
+  void get_db_statistics(Formatter *f);
```

Comment:
might want to make this method with `override`. 

nit, could move `*` right after `Formatter`, as `*` is part of the type.

Reply:
Ok，changed it